interface/forms/clinical_notes/templates/report.html.twig

```diff
@@ -14,15 +14,15 @@ The Report template of the clinical note form.
             <header class="d-flex justify-content-between bg-light border border-dark p-1 sticky-top">
                 <div class="form-group mb-0">
                     <div class="font-weight-bold">{{ "Date"|xlt }}</div>
-                    <time datetime="{{ n.date|date()|text }}">{{ n.date|date()|text }}</time>
+                    <time datetime="{{ n.date|date()|attr }}">{{ n.date|date()|text }}</time>
                 </div>
                 <div class="form-group mb-0">
                     <div class="font-weight-bold">{{ "Type"|xlt }}</div>
-                    <div>{{ n.codetext|default("Unspecified"|xl)|xlt }}</div>
+                    <div>{{ n.codetext is defined ? n.codetext|xlListLabel|text : "Unspecified"|xlt }}</div>
```

Comment:
Generally I am a big fan of ternary operators, and while this code will get to the same end, the following would be a cleaner solution
```
{{ n.codetext|xlListLabel|default("Unspecified"|xlt) }}
``

It is more "Twig-ish" in its processing. The only caveat would be if `xlListLabel` interferes with `default()` - I'd have to test.

Reply:
The issue with that solution is if the translation ends up taking a string and making it empty, then it will erroneously set the title of that value to Unspecified rather than just making it blank (this type of thing should not happen, but if it does the ternary operator would avoid that weird scenario).

Reply:
i wonder if `is defined` will cover everything though ...? (ie. also need to check if empty etc. ?)

Reply:
turns out need to use `is not empty`